### PR TITLE
Update color-schemes.drconf || New theme: Gruvbox

### PR DESCRIPTION
**Pull request by SparkScratch-P**
## A New theme, named Gruvbox has been added. Pls see to it, will be happy to be merged!

### CODE REVIEW

**Review by Gusted — APPROVED** [review]
LGTM

File: src/config/color-schemes.drconf [hunk 1]
```diff
@@ -39,3 +39,15 @@ text: #93a1a1
 LIGHT
 background: #fdf6e3
 text: #586e75
+
+================================
+
+Gruvbox
```
Comment (Gusted) [inline src/config/color-schemes.drconf:45]: This entry isn't in alphabetic order.

**Review by Gusted — CHANGES_REQUESTED** [review]

**Comment by SparkScratch-P** [comment]
## I have done the required changes. Please merge this pull.

**Review by Gusted — APPROVED** [review]

**Review by dhasial — COMMENTED** [review]
I would suggest changing the dark text colour to #c8ba99 which would make it much easier to read text since while still fitting the gruvbox dark aesthetic

**Pull request merged**